cli/cmd/root.go

```diff
@@ -27,12 +28,35 @@ Open source data integration at scale.
 Find more information at:
 	https://www.cloudquery.io`
 
-	disableSentry   = false
-	analyticsClient *AnalyticsClient
-	logFile         *os.File
-	invocationUUID  uuid.UUID
+	disableSentry      = false
+	oldAnalyticsClient *AnalyticsClient
+	logFile            *os.File
+	invocationUUID     uuid.UUID
 )
 
+func wrapRunE(cmd *cobra.Command) {
+	// Ideally we could use PersistentPostRunE, but it doesn't get called if the command has an error
+	// See https://github.com/spf13/cobra/issues/1893
+	if cmd.RunE != nil {
+		wrapped := cmd.RunE
+		cmd.RunE = func(cmd *cobra.Command, args []string) error {
+			commandName := cmd.Name()
+			if cmd.Parent() != nil {
```

Comment:
Not sure if it's worth handling more command nesting (we have only two levels, e.g. `plugin install`)